### ISSUE TITLE
Pre-trained model 

### ISSUE THREAD

**Issue by ChengyaoLi** [xinshuoweng/mono3D_PLiDAR#2]
Thanks for planing to make this code public. Before the code goes fully public, is it possible for you to share the Mask RCNN model (either the one before fine-tuned on KITTI or after)?
